index.html

```diff
@@ -103,7 +103,7 @@ <h1>MUHYIDEEN ELIAS</h1>
   </section>
   <section class="section-4">
     <div class="my-recent-work">
-      <h4>MY RECENT WORK</h4>
+      <h2>MY RECENT WORK</h2>
     </div>
     <div class="breakage">
       <hr>
```

Comment:
- [ ] @team great work so far, I am having difficulty navigating the page using only my keyboard, some of your `<a>` tags are not there, especially in social media icons and the work section, kindly fix it.